lib/src/main/java/graphql/nadel/engine/transform/hydration/batch/NadelBatchHydrationInputBuilder.kt

```diff
@@ -54,8 +55,15 @@ internal object NadelBatchHydrationInputBuilder {
                     }
                     // These are batch values, ignore them
                     is NadelHydrationActorInputDef.ValueSource.FieldResultValue -> null
+                    is NadelHydrationActorInputDef.ValueSource.StaticValue -> {
+                        val staticValue: NormalizedInputValue = makeNormalizedInputValue(
+                            type = actorFieldArg.actorArgumentDef.type,
+                            value = valueSource.value,
+                        )
+                        actorFieldArg to staticValue
+                    }
                 }
-            },
+            }
```

Comment:
```suggestion
            },
```

Nit: keep the trailing comma